stories/index.js

```diff
@@ -128,6 +128,15 @@ storiesOf('Animation', module)
     <PieChart data={dataMock} lengthAngle={-360} animate />
   ));
 
+storiesOf('Filtering Segments', module).add('Ignore orange Segment', () => (
+  <PieChart
+    data={dataMock}
+    filterSegments={(dataEntry, index) => {
+      return dataEntry.color === '#E38627';
+    }}
+  />
+));
+
```

Comment:
No need to write a storybook example for this since this feature is a very technical one. Would you mind removing this? :)